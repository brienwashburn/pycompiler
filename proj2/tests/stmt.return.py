return
return a
return (a, b, c)
return a,
return foo(10, *(foo if bar else quux))
