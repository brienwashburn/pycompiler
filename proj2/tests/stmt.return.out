(Module
 (Return)
 (Return (Name a))
 (Return (Tuple (Name a) (Name b) (Name c)))
 (Return (Tuple (Name a)))
 (Return
  (Call
   (func (Name foo))
   (args (Num 10))
   (keywords)
   (starargs (IfExp (Name bar) (Name foo) (Name quux)))
   (kwargs #f))))
